internal/apply/apply.go

```diff
@@ -41,21 +50,26 @@ Only a single environment may be suitable for an initial test, but for a real sy
 
 	flog.Infof("Infrastructure executor: %s", "Terraform")
 
-	applyAll(rootDir, *projectConfig, environments)
+	err = modulesWalkCmd("apply", rootDir, projectConfig, []string{"make"}, environments)
+	if err != nil {
+		return errors.New(fmt.Sprintf("Module Apply failed: %s", err.Error()))
+	}
 
 	flog.Infof(":check_mark_button: Done.")
 
-	summarizeAll(rootDir, *projectConfig, environments)
+	flog.Infof("Your projects and infrastructure have been successfully created.  Here are some useful links and commands to get you started:")
+	err = modulesWalkCmd("summary", rootDir, projectConfig, []string{"make", "summary"}, environments)
+	if err != nil {
+		return errors.New(fmt.Sprintf("Module summary failed: %s", err.Error()))
+	}
+	return err
 }
 
-func applyAll(dir string, projectConfig projectconfig.ZeroProjectConfig, applyEnvironments []string) {
-	environmentArg := fmt.Sprintf("ENVIRONMENT=%s", strings.Join(applyEnvironments, ","))
-
+func modulesWalkCmd(lifecycleName string, dir string, projectConfig *projectconfig.ZeroProjectConfig, cmdArgs []string, applyEnvironments []string) error {
```

Comment:
Maybe `applyEnvironments` should be `environments` now that it's not apply specific??